### ISSUE TITLE
BBC download at a glacial speed and ITV stalls out every five minutes. This is on all three of my Macs. 

### ISSUE THREAD

**Issue by notascot** [Ascoware/get-iplayer-automator#309]
**This issue tracker is not a discussion forum, so don't use it as such.  If you wish to discuss the BBC, iPlayer or anything else that is not related to an actual problem with the current release of GiA, post in an appropriate forum, not in this issue tracker.**

**STOP! DO NOT submit a ticket if you are reporting that you are unable to download a program from outside the UK**.

If you need an explanation, read [this](https://github.com/Ascoware/get-iplayer-automator/wiki/No-Support-for-GiA-Outside-the-UK). If you ignore this instruction, your ticket will be closed with no further action. You can fib about your location if you like, but you will only be wasting your time. If you are using GiA within the UK, continue with the instructions below.

Before creating a new ticket, search the current open and closed tickets to see if your issue has already been reported. Use the search box at the top of the page to do so. You can also use the links below to view open and closed tickets in list form.

#### [Closed tickets](https://github.com/Ascoware/get-iplayer-automator/issues?q=is%3Aissue+is%3Aclosed)

If you create a ticket for an issue that has already been reported, your ticket will be closed with no further action. If you are certain your issue has not yet been reported, continue with the instructions below.

Create a succinct title describing your issue in the `Title` field above, then provide the information requested below. If you do not provide the requested information, your ticket will be closed with no further action.

If you cannot download a program, provide a verbose log of the attempt. See [Verbose Log](https://github.com/Ascoware/get-iplayer-automator/wiki/Verbose-Log) in wiki for instructions on posting a verbose log for your ticket. **Your verbose log should show the results from downloading ONLY ONE program. DO NOT post a massive log covering multiple multiple program downloads. IMPORTANT: DO NOT paste your verbose log into your issue report. If you do, expect it to be deleted.**

If GiA crashes, provide the OS X crash log by copying the information from Crash Reporter into a file or locating the crash log in `$HOME/Library/Logs/DiagnosticReports`. See [Verbose Log](https://github.com/Ascoware/get-iplayer-automator/wiki/Verbose-Log) in wiki for instructions on posting a crash log for your ticket. **IMPORTANT: DO NOT paste your crash log into your issue report. If you do, expect it to be deleted.**

**Delete the instructions above before submitting your ticket.**

---------------

*What steps will reproduce the problem? Be specific, and provide as much detail as possible.*

*What is the expected output? What do you see instead?*

*What version of GiA are you using? On what version of OS X?*

*Are you using a web proxy? DNS proxy? VPN? VPS?*

*What program are you attempting to download (full title, series and episode)? **Provide only ONE program as an example.** *

*Please provide any additional information below.*

**Comment by skovatch** [comment]
I'm not sure what I have to offer here. Slow BBC downloads could be anything from a busy network connection to a slow VPN link. Same for ITV.

**Comment by notascot** [comment]
Thanks Scott as always for your response,

What stumped me was the fact that I have a blazing fast wifi connection, and the fact that that if I download the same BBC show directly from  BBC iPlayer it downloads way faster.  As to ITV shows, I have no explanation for what happens there. The downloads simply stall multiple times during download.

Thank you for your continuing work on Get iPlayer and the personal support you provide.

Stay safe and stay healthy,

Phillip John Davies

**Comment by Biffothebear666** [comment]
I have the same issue, Speedtest shows 150M up and 50M down  but when downloading I only get 100kb/s. starting a download elsewhere sees the download  speed. ramping back to normal? So the network side is OK
I'm using a VPN ( different VPN's produce the same symptom) and do get log messages of:
ERROR: Response: 403 Forbidden
ERROR: Access to this resource was blocked by the BBC
ERROR: Ignore this error if programme download is successfu
 
but the download is OK, it is sticking somewhere and waiting to timeout before the next block?
